.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "net_handler/protos"]
```

Comment:
Should we add a comment here about submodules or make it a section in the wiki?

Reply:
Yea I'll add a section describing submodules in the wiki, maybe on the Git page or something?